src/commands/file.lisp

```diff
@@ -103,6 +104,20 @@
     (encoding-read-error ()
       (open-external-file pathname))))
 
+(define-command find-file-directory () ()
+  "Open this file's directory and place point on the filename."
+  (let ((fullpath (buffer-filename)))
+    (cond
+      ((null fullpath)
+       (message "No file at point"))
+      (t
+       (switch-to-buffer
+        (find-file-buffer (lem-core/commands/file::directory-for-file-or-lose (buffer-directory))))
+       (let ((filename (file-namestring fullpath)))
+         (search-forward (current-point) filename)
+         (window-recenter (current-window))
+         (character-offset (current-point) (* -1 (length filename))))))))
```

Comment:
How about making this sequence of operations a function of the directory-mode package?